ros__parameters:
  axis_angular:
    pitch: 0
    roll: 1
    yaw: 2
  axis_linear:
    x: 4
    y: 3
    z: 5
  enable_button: 0
  enable_turbo_button: 1
  require_enable_button: false
  scale_angular:
    pitch: -0.05
    roll: -0.05
    yaw: 0.05
  scale_angular_turbo:
    pitch: -0.1
    roll: -0.1
    yaw: 0.1
  scale_linear:
    x: -0.05
    y: -0.05
    z: -0.05
  scale_linear_turbo:
    x: -0.2
    y: -0.2
    z: -0.2
